js/private/js_binary.sh.tpl

```diff
@@ -232,9 +257,14 @@ if [ "${node:0:1}" = "/" ]; then
         exit 1
     fi
 else
-    export JS_BINARY__NODE_BINARY="$JS_BINARY__RUNFILES/{{node}}"
+    if [ "${JS_BINARY__NO_RUNFILES:-}" ]; then
+        export JS_BINARY__NODE_BINARY
```

Comment:
What is the reason for exporting before doing the assignment? Not doing the export+assignment on the same line?

Reply:
Its a shell check warning to assign `$(...)` to a variable on the same line as exporting it.
```
export FOO=$(bar)
```

Reply:
I see, thanks 👍 

IMO it's odd when we flip back and forth though. Now all within ~10 lines of code we have:
```
export X
X = a
export X=b
```

Can we only export it once at the top (above the `if`) or something like that? Doesn't have to be now, just a thought for next time we change this (or similar)...